rita_bin/src/extender.rs

```diff
@@ -0,0 +1,75 @@
+//! This is the main source file for the Rita extender binary
+
+#![warn(clippy::all)]
+#![allow(clippy::pedantic)]
+#![forbid(unsafe_code)]
+
+#[cfg(feature = "jemalloc")]
+use jemallocator::Jemalloc;
+#[cfg(feature = "jemalloc")]
+#[global_allocator]
+static GLOBAL: Jemalloc = Jemalloc;
```

Comment:
we really don't there was a time there when jemalloc was often faster than the system allocator and I thought it might matter, but it can be removed from all crates.